services/interface.go

```diff
@@ -1,7 +1,10 @@
 package services
 
-// TODO Finish service interface and implement
+import "github.com/labstack/echo/v4"
+
 // Service aims to help autodiscover our avialable services
+// TODO Finish service interface and implement
 type Service interface {
-	GetServiceName() string
+	// GetHandlers returns a map of GET endpoint strings followed by the representative handler function via the echo HandlerFunc interface
+	GetHandlers() map[string]echo.HandlerFunc
```

Comment:
Do we want to break up the handlers by HTTP verb? Or should we just autopopulate all of them with this method?

Reply:
Ideally it would just be one, I don't have an informed enough view with this one service as to what that looks like exactly.